ttnn/cpp/ttnn/operations/data_movement/common/common.hpp

```diff
@@ -156,6 +156,27 @@ ttnn::Tensor pad_to_tile_vol(
     const bool use_multicore,
     const std::optional<MemoryConfig>& memory_config);
 
+enum class ShardStrategy { BLOCK, HEIGHT, WIDTH };
```

Comment:
isn't tis enum already defined in a different place?

Reply:
No, only in the ttnn Python code. It would be good to find it a better home later on; perhaps under tt::tt_metal?